src/Symfony/Component/HttpFoundation/Tests/RequestTest.php

```diff
@@ -330,6 +330,16 @@ public function testGetMimeTypeFromFormat($format, $mimeTypes)
         }
     }
 
+    /**
+     * @dataProvider getFormatToMimeTypeMapProvider
+     */
+    public function testGetMimeTypesFromFormat($format, $mimeTypes)
+    {
+        if (null !== $format) {
+            $this->assertEquals($mimeTypes, Request::getMimeTypes($format));
+        }
+    }
+
```

Comment:
I think one more test could be useful, which asserts what *getMimeTypes()* returns, if the format is **not** found in *Request::$formats*.